CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.1.1] - 2022-10-07
+
+### Added
+- Wallet handling via CRUD
+- Verifiable Credentials and Verifiable Presentation handling via CRUD
+- Revocation of Verifiable Credentials
+- DID document and Service Endpoint handling via CRUD
```

Comment:
Should mention/distinguish managed ans self-managed wallets (for the latter, only basic handling and interaction is implemented)